package/gluon-mesh-vpn-wireguard/luasrc/lib/gluon/upgrade/400-mesh-vpn-wireguard

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/lua
+
+local uci = require('simple-uci').cursor()
+local site = require 'gluon.site'
+
+local private_key = uci:get("network", "vpn", "private_key")
+
+if not private_key or not private_key:match("^" .. ("[%a%d+/]"):rep(43) .. "=$") then
```

Comment:
`^[A-Za-z0-9+/]{42}[A|E|I|M|Q|U|Y|c|g|k|o|s|w|4|8|0]=$` is more accurate: https://lists.zx2c4.com/pipermail/wireguard/2020-December/006222.html